ykdl/compact.py

```diff
@@ -18,6 +18,8 @@
     compact_bytes = bytes
     from urllib.parse import unquote as compact_unquote
     from urllib.parse import quote
+    import socks
+    from sockshandler import SocksiPyHandler
```

Comment:
I am talking about urlparse, not socks, if you has no plan to fix `SocksiPyHandler` yourself.

BTW: same import should put outside of py2/3 judge.